netci.groovy

```diff
@@ -3,14 +3,36 @@ import jobs.generation.ArchivalSettings;
 
 def project = GithubProject;
 def branch = GithubBranchName;
-loggingOptions = "/clp:v=detailed /p:MinimalConsoleLogOutput=false";
+loggingOptions = "";
 
 def addArchival(def job) {
   def archivalSettings = new ArchivalSettings()
-  archivalSettings.addFiles("src/**/artifacts/**/log/*.binlog")
+  archivalSettings.addFiles("bin/logs/*")
+  // also grab prebuilt reports
+  archivalSettings.addFiles("bin/logs/**/*")
+  archivalSettings.addFiles("src/**/*.binlog")
+  archivalSettings.addFiles("src/**/*.log")
+  archivalSettings.addFiles("init-tools.log")
+  archivalSettings.addFiles("msbuild.log")
   archivalSettings.addFiles("testing-smoke/smoke-test.log")
+  // tarball builds use subdirectories, so add these
+  archivalSettings.addFiles("source-build/bin/logs/*")
+  archivalSettings.addFiles("source-build/bin/logs/**/*")
+  archivalSettings.addFiles("source-build/src/**/*.binlog")
+  archivalSettings.addFiles("source-build/src/**/*.log")
+  archivalSettings.addFiles("source-build/init-tools.log")
+  archivalSettings.addFiles("source-build/msbuild.log")
+  archivalSettings.addFiles("source-build/testing-smoke/smoke-test.log")
+  archivalSettings.addFiles("tarball-output/bin/logs/*")
+  archivalSettings.addFiles("tarball-output/bin/logs/**/*")
+  archivalSettings.addFiles("tarball-output/src/**/*.binlog")
+  archivalSettings.addFiles("tarball-output/src/**/*.log")
+  archivalSettings.addFiles("tarball-output/init-tools.log")
+  archivalSettings.addFiles("tarball-output/msbuild.log")
+  archivalSettings.addFiles("tarball-output/testing-smoke/smoke-test.log")
```

Comment:
Maybe you're just in the middle of trying things out, but I suggest deduping with something like:

```groovy
["source-build", "tarball-output"].each { logRoot ->
```

Reply:
Thanks, I will definitely clean this up before checking in.